database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Sanctum migrations are no longer auto-migrated, so the migration needed to be published.